src/morph/src/Server.re

```diff
@@ -1,8 +1,17 @@
-type handler = Opium_core.Service.t(Request.t, Response.t);
+type handler('req_body, 'res_body) =
```

Comment:
I removed `Opium_core` here since we really don't need it, it's a very small implementation

Reply:
cool! No opium dependency ?

Reply:
Yepp, it is now completely removed. I just want to move the handler/middleware to their own module.